.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  ci:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+
+    - name: Install requirements (python)
+      run: pip3 install pytest
+
+    - name: Setup up PYTHONPATH
+      run: echo "PYTHONPATH=${PWD}/src" >> $GITHUB_ENV
```

Comment:
Can you just use `pip install .`? Messing with the `PYTHONPATH` environment variable is never a good idea.

Reply:
Sure, let me fix that.